src/libraries/Microsoft.PowerFx.Connectors/Shared/ServiceFunction.cs

```diff
@@ -46,18 +47,27 @@ internal sealed class ServiceFunction : BuiltinFunction, IAsyncTexlFunction2
         private readonly WeakReference<IService> _parentService;
         private readonly string _actionName;
         private readonly bool _numberIsFloat;
+        private readonly string _pageLink;
+        private readonly bool _isDeprecated;
```

Comment:
>_isDeprecated [](http://example.com/codeflow?start=30&length=13)

nt: can we comment this one: 
- it's still callable, 
- but we issue a warning if used 
- hidden from  intellisense

This also helps explain the different with "supported":
- not callable
- shown in intellisense with error